data/DCL/projects.yaml

```diff
@@ -288,30 +350,56 @@ projects:
         language: Python / C++
         information:
             - type: Paper
-              url: https://mlsys.org/Conferences/2019/doc/2019/54.pdf
-              title: "AggregaThor: Byzantine Machine Learning via Robust Gradient Aggregation (SysML19')"
+              url: https://infoscience.epfl.ch/record/265684
+              title: "AGGREGATHOR: Byzantine Machine Learning via Robust Gradient Aggregation"
```

Comment:
I don't want to get into a typography war :-), but shouldn't the large and small caps used in the paper translate to upper and lower case when using a font without small caps?

Reply:
Hoo, you're right, I copied it from the summary page on infoscience.epfl.ch, but didn't looked directly at the paper.

PS: typography war are one of the few worth fighting, and you easily won this one :)